en/docs/learn/self-registration-and-account-confirmation.md

```diff
@@ -220,7 +220,24 @@ Next, you can try out self-registration.
 
 3.  Fill in the user details, provide consent to share the requested
     information and then click **Register**.
+    ![self-signup-form](../assets/img/using-wso2-identity-server/self-signup-form.png) 
+        
+    ??? Abstract "Click here to see how to configure above requested attributes for the self registration" 
+        
+        -   Attributes that shows in the
+        self sign up page are the WSO2 [local dialect's](../../learn/adding-claim-mapping/#add-local-claim) claims which has
+        **Supported by Default** configuration enabled. 
+        -   Navigate to claim
+        dialect list page and go to **http://wso2.org/claims** dialect and
+        expand any claim to view the configuration. 
```

Comment:
Click **List** under **Claims** and select the **http://wso2.org/claims** dialect. Expand any claim to view the configuration.